service-11/PUS-11-13.asn1

```diff
@@ -0,0 +1,52 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-11-13 DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    Subschedule-ID FROM Subschedule
+    Group-ID FROM Group
+    PUSC-UINT32, AbsoluteTime FROM BasicTypes
+    ST-11-Request-ID FROM ST-11-Request;
+
+maxTM-11-13-ReportCount INTEGER ::= 4095
+
+TM-11-13-TimebasedScheduleSummaryReport ::= SEQUENCE
+{
+--! n UnsignedInteger, moved to acn
+    report SEQUENCE (SIZE(0 .. maxTM-11-13-ReportCount)) OF TM-11-13-Report
+}
+
+TM-11-13-Report ::= TM-11-13-ReportGeneric {Subschedule-ID, Group-ID, AbsoluteTime, ST-11-Request-ID}
```

Comment:
`OPTIONAL`s should be enabled or disabled in default implementation (or encoding will be incorrect - additional bits will be added to mark fields as present or not)

use `WITH COMPONENTS...` construct